src/styles/content/tables/styles.less

```diff
@@ -439,6 +439,7 @@
 }
 
 .table-content-spacing-left {
+  display: inline-block;
```

Comment:
this is a workaround for Tooltip rendering a nested `div`-structure. i'm not saying we should address this now, but `.table-content-spacing-left` now definitely does more than its name is suggesting.

we could have added a class that keeps the Tooltip in check to keep things separated.